tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -3290,26 +3292,29 @@ std::string hopperTestName(
 }
 
 std::string hopperTestNameSwizzle(
-    const testing::TestParamInfo<HopperMatmulSchedulerTestParams>& info) {
+    const testing::TestParamInfo<HopperPlusMatmulSchedulerTestParams>& info) {
   std::unordered_map<MmaMacro, std::string> mma_macro_to_swizzle_str_map = {
       {MmaMacro::Hopper_64_256_16, "128BSwizzle"},
       {MmaMacro::Hopper_64_128_16, "128BSwizzle"},
       {MmaMacro::Hopper_64_64_16, "128BSwizzle"},
       {MmaMacro::Hopper_64_32_16, "64BSwizzle"},
-      {MmaMacro::Hopper_64_16_16, "32BSwizzle"}};
+      {MmaMacro::Hopper_64_16_16, "32BSwizzle"},
+      {MmaMacro::Blackwell_128_256_16, "128BSwizzle"},
+      {MmaMacro::Blackwell_128_128_16, "128BSwizzle"},
+      {MmaMacro::Blackwell_128_64_16, "128BSwizzle"},
+      {MmaMacro::Blackwell_128_32_16, "64BSwizzle"},
+      {MmaMacro::Blackwell_128_16_16, "32BSwizzle"}};
```

Comment:
```suggestion
      {MmaMacro::Blackwell1CTA_128_256_16, "128BSwizzle"},
      {MmaMacro::Blackwell1CTA_128_128_16, "128BSwizzle"},
      {MmaMacro::Blackwell1CTA_128_64_16, "128BSwizzle"},
      {MmaMacro::Blackwell1CTA_128_32_16, "64BSwizzle"},
      {MmaMacro::Blackwell1CTA_128_16_16, "32BSwizzle"}};
```
**[Suggested by AI]** Verified that the build will pass with this change.